contrib/babelfishpg_tds/src/backend/tds/tdslogin.c

```diff
@@ -920,6 +920,11 @@ FetchLoginRequest(LoginRequest request)
 
 		if (request->sspiLen > 0)
 		{
+			bool is_windows_allowed = GetConfigOptionFlags("babelfishpg_tsql.pltsql_is_windows_allowed", true);
+			if (!is_windows_allowed)
+				ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+						errmsg("SSPI authentication is not supported in babelfish")));
```

Comment:
kerberos authentication is not supported.

Reply:
Done!